### PR TITLE
hotfix: amend bribe processor script

### PR DESCRIPTION
**Pull request by jorijnsmit**
slight changes i had to make to make to successfully interact with the bribes processor contract.

### CODE REVIEW

File: scripts/badger/process_bribes.py [hunk 1]
```diff
@@ -99,22 +103,51 @@ def step1():
...
+        PROCESSOR.swapCVXTobveCVXAndEmit()
+    if BADGER.balanceOf(PROCESSOR) > 0:
+        PROCESSOR.emitBadger()
+    SAFE.post_safe_tx(call_trace=True)
+
+
+def step3_a():
+    '''can be skipped if step3 was successful'''
+
+    if CVX.balanceOf(SAFE) > 0:
```
Comment (petrovska-petro) [inline scripts/badger/process_bribes.py:143]:
```suggestion
    if CVX.balanceOf(PROCESSOR) > 0:
```

File: scripts/badger/process_bribes.py [hunk 2]
```diff
@@ -99,22 +103,51 @@ def step1():
...
+        PROCESSOR.swapCVXTobveCVXAndEmit()
+    SAFE.post_safe_tx(call_trace=True)
+
+
+def step3_b():
+    '''can be skipped if step3 was successful'''
+
+    if BADGER.balanceOf(SAFE) > 0:
```
Comment (petrovska-petro) [inline scripts/badger/process_bribes.py:151]:
```suggestion
    if BADGER.balanceOf(PROCESSOR) > 0:
```

**Review by petrovska-petro — DISMISSED** [review]
LGTM! 

good one leverage `isClaimed` in the votium merkle contract rather than catching all exceptions 💪

**Comment by jorijnsmit** [comment]
will need some more changes due to the new bribe processor; will commit here.

**Comment by jorijnsmit** [comment]
waiting until we are able to test in prod before moving away from draft pr status.

**Comment by jorijnsmit** [comment]
as used to solve #473

**Review by petrovska-petro — APPROVED** [review]
- addresses for eurs and usdn looks correct 

- 3h deadline change convo [ref](https://discord.com/channels/869166959739170836/935460632818516048/980853102930448474) for other reviewers

**Review by petrovska-petro — APPROVED** [review]

**Review by jalbrekt85 — APPROVED** [review]
lgtm

**Pull request merged**